### PR TITLE
avoid crashing on unhandled message type

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: shipwright/cli.py [hunk 1]
```diff
@@ -251,4 +251,4 @@ def switch(rec):
   elif rec['event'] == 'removed':
     return 'Untagging {image}:{tag}'.format(**rec)
   else:
-    return rec
```
Comment (graingert) [inline shipwright/cli.py]: if rec is passed straight through here, you try to print a dict.

```
Traceback (most recent call last):
  File "/home/graingert/.virtualenvs/shipwright/bin/shipwright", line 9, in <module>
    load_entry_point('shipwright==0.3.0', 'console_scripts', 'shipwright')()
  File "/home/graingert/.virtualenvs/shipwright/local/lib/python2.7/site-packages/shipwright/cli.py", line 195, in main
    show_fn(writer(event))
  File "/home/graingert/.virtualenvs/shipwright/local/lib/python2.7/site-packages/shipwright/cli.py", line 228, in highlight_
    print(color_fn(name) + " | " + msg)
TypeError: coercing to Unicode: need string or buffer, dict found
```

File: tests/test_unhandled_record.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from shipwright.cli import switch
+
+
+example = {
```
Comment (graingert) [inline tests/test_unhandled_record.py]: I got a record that looks like this, it wasn't handled.

I'd rather just see the JSON dumps of this, but if you think it should print something else, that would be cool too.

File: tests/test_unhandled_record.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    u'aux': {
+        u'Digest': u'sha256:redacted',
+        u'Size': 1337,
+        u'Tag': u'redacted'
+    },
+   'event': 'push',
+   'image': u'redacted/redacted',
+   u'progressDetail': {}
+}
```
Comment (graingert) [inline tests/test_unhandled_record.py]: I got a record that looks like this, it wasn't handled.

I'd rather just see the JSON dumps of this, but if you think it should print something else, that would be cool too.

**Comment by srobertson** [comment]
@graingert rebase master, if tests pass I'll accept

**Pull request merged**